util.go

```diff
@@ -2,26 +2,16 @@ package main
 
 import (
 	"fmt"
-	"log"
-	"strings"
 
 	"github.com/gogo/protobuf/jsonpb"
-	"github.com/gogo/protobuf/proto"
 	authnpb "istio.io/api/authentication/v1alpha1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/json"
-	"sigs.k8s.io/yaml"
-)
 
-type objectStruct struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              map[string]interface{} `json:"spec,omitempty"`
-}
+	convertpkg "github.com/istio-ecosystem/security-policy-migrate/converter"
+)
 
-func convertToPolicy(item unstructured.Unstructured) (*InputPolicy, error) {
+func convertToPolicy(item unstructured.Unstructured) (*convertpkg.InputPolicy, error) {
```

Comment:
why not just move this whole file to converter/util.go?

Reply:
Done